static/js/common.ts

```diff
@@ -136,9 +136,12 @@ export function setup_password_visibility_toggle(
 ): void {
     const label = $t({defaultMessage: "Show password"});
     set_password_toggle_label(password_selector, label, tippy_tooltips);
-    $(password_selector).on("click", (e) => {
-        e.preventDefault();
-        e.stopPropagation();
-        toggle_password_visibility(password_field_id, password_selector, tippy_tooltips);
-    });
+    $(password_selector).on(
+        "click",
+        (e: {preventDefault: () => void; stopPropagation: () => void}) => {
```

Comment:
This is incorrectly typed. It should be `JQuery.ClickEvent<HTMLElement, undefined, HTMLElement, HTMLElement>`. Furthermore, this is not implicitly `any` but the type I mentioned above so this is not required.  

You can verify by calling `e.random()` and running `tools/lint --modified` and it will give a correct error:
```
tsc       | static/js/common.ts(140,11): error TS2339: Property 'random' does not exist on type 'ClickEvent<HTMLElement, undefined, HTMLElement, HTMLElement>'.
```